extern/boostd-data/Dockerfile.test

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.18-alpine
+FROM golang:1.21-alpine
```

Comment:
I think default is v1.20 for Golang (based on Lotus). Shall we stick to that?

Reply:
I don't think it matters much -- this docker stack is used solely for testing.